app/assets/stylesheets/common/modal/modal-overrides.scss

```diff
@@ -310,11 +310,7 @@
 .d-modal.fullscreen-table-modal {
   .d-modal {
     &__container {
-      max-width: $reply-area-max-width;
       max-height: unset;
     }
-    &__header {
-      justify-content: flex-end;
```

Comment:
This was only needed to put the X to the right of the top of the modal, if the modal title is provided this is not necessary

Reply:
Will it always be provided? That would be good, if not, I have another solution.

Reply:
I don't think so, it is possible to not provide it, so yeah we should probably handle those cases too :)

Reply:
done